x-pack/plugin/watcher/src/main/java/org/elasticsearch/xpack/watcher/actions/index/IndexAction.java

```diff
@@ -29,7 +29,7 @@
 
     public static final String TYPE = "index";
 
-    @Nullable final String docType;
```

Comment:
The changes to this file (IndexAction) were missed in https://github.com/elastic/elasticsearch/pull/37594. I added it here to help us find what to remove when we completely get rid of types. Apologizes for conflating the two issues.